### PR TITLE
Scoreboard UI

### PR DESCRIPTION
**Pull request by kesang20**
Added styling to the scoreboard

### CODE REVIEW

File: src/game/ApplesBoardComponents/PlayerView.js [hunk 1]
```diff
@@ -48,7 +48,7 @@ export default function PlayerView({
     setChatInput(value);
   }
   let answers;
-  if (G.activePrompt.body) {
+  if (G.activePrompt.text) {
```
Comment (CotyJ) [inline src/game/ApplesBoardComponents/PlayerView.js]: Should be .body

File: src/game/ApplesBoardComponents/PlayerView.js [hunk 2]
```diff
@@ -60,7 +60,7 @@ export default function PlayerView({
               playerId={i}
               moves={moves}
               key={card.id}
-              text={card.body}
+              text={card.text}
```
Comment (CotyJ) [inline src/game/ApplesBoardComponents/PlayerView.js]: Should be .body

File: src/game/ApplesBoardComponents/PlayerView.js (hunk 1 above)
Comment (crypto-bender) [inline src/game/ApplesBoardComponents/PlayerView.js]: database keys are still body, probably need to swap back to body.

**Review by CotyJ — APPROVED** [review]
Looks good!

**Review by crypto-bender — APPROVED** [review]
Looks good

**Review by codingavatar — APPROVED** [review]
Looks good!

**Pull request merged**